File: Problem Set 1 (Objective-C)/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){

    int height;

    do
    {
     height = get_int("Height: ");
    }
    while (height<1 || height > 8);

    for (int a = 1; a <= height; a++)
    {
         for(int x=0; x < height-a; x++)
         {
             printf(" ");
         }

         for (int b = 0; b < a; b++)
          {
              printf("#");
          }

          printf("\n");
    }
}

File: Problem Set 6 (Python)/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(1,height+1,1):

    for z in range(0,height-i,1):
        print(" ", end ="")

    for x in range(0,i,1):
        print("#", end ="")

    print("\n",end="")